c/column_from_pylist.cc

```diff
@@ -255,21 +253,21 @@ static bool parse_as_str(PyyList& list, MemoryRange& offbuf,
   T curr_offset = 0;
   size_t i = 0;
   for (i = 0; i < nrows; ++i) {
-    PyObj item = list[i];
+    py::obj item = list[i];
 
     if (item.is_none()) {
       offsets[i] = curr_offset | GETNA<T>();
       continue;
     }
     if (item.is_string()) {
-      size_t len = 0;
-      const char* cstr = item.as_cstring(&len);
-      if (len) {
-        T tlen = static_cast<T>(len);
+      CString cstr = item.to_cstring();
```

Comment:
First thought this is the MS `CString` class and was not sure how it is going to work :)